src/server/report/app.js

```diff
@@ -114,7 +112,13 @@ class App extends LitElement {
 				if (!testData) {
 					view = html`<p>Test not found: <b>${this._filterTest}</b>.</p>`;
 				} else {
-					view = this._renderTest(fileData, testData);
+					hasPadding = false;
+					view = html`
+						<d2l-vdiff-report-test full-mode="${this._fullMode}" file="${fileData.name}" layout="${this._layout}" ?show-overlay="${this._overlay}" test="${testData.name}" @setting-change="${this._handleSettingChange}"></d2l-vdiff-report-test>
+						<div class="padding">
```

Comment:
I think this back button will eventually live in that panel once I add next/previous iterators.

Reply:
If we don't move it to the header, the button seems to jump around a lot as you between full and split mode. I wonder if we could preserve the space so it stays put.

But if we do move it to the header, not an issue!

Reply:
It's going to the header for sure, but in the next PR where I add tabs for the different tabs it'll jump around a lot less!